recipe/meta.yaml

```diff
@@ -22,12 +22,12 @@ requirements:
     - python
     - poetry-core 1.5
```

Comment:
These packages are redundant if the build backend is poetry:
```
    - setuptools
    - wheel
```

Reply:
removed!